.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+# What's in this PR?
+
+# Why 
+
+# Whats Next?
```

Comment:
What's next should be the last section, how this was tested should be before that